packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js

```diff
@@ -92,15 +93,65 @@ export default function InspectedElementWrapper(_: Props) {
     (canViewElementSourceFunction === null ||
       canViewElementSourceFunction(inspectedElement));
 
+  const isErrored = inspectedElement != null && inspectedElement.isErrored;
+
   const isSuspended =
     element !== null &&
     element.type === ElementTypeSuspense &&
     inspectedElement != null &&
     inspectedElement.state != null;
 
+  const canToggleError =
+    inspectedElement != null && inspectedElement.canToggleError;
+
   const canToggleSuspense =
     inspectedElement != null && inspectedElement.canToggleSuspense;
 
+  const toggleErrored = useCallback(() => {
+    let nearestErrorBoundary = null;
+    let currentElement = element;
+    while (currentElement !== null) {
+      if (currentElement.isErrorBoundary) {
+        nearestErrorBoundary = currentElement;
```

Comment:
This logic should be moved into the backend I think, as part of inspecting a Fiber. That way we don't have to _eagerly_ send the `isErrorBoundary` bit for every Fiber that's mounted.

One other small thing I noticed while testing this that I think we should change:

If I select an error boundary component and click the toggle, the error boundary _catches its own error_ which is not how it would work in React itself. I worry this might give people the incorrect impression that a boundary can catch its own errors when in reality, boundaries only catch errors that happen _beneath them_ in the tree.

So I think the fix here would be for the backend to always start the error boundary check at `inspectedElement.return` (rather than `inspectedElement` itself). Does this make sense?